service/src/main/resources/templates/ehr_encounter_to_ehr_composition_template.mustache

```diff
@@ -1,19 +1,31 @@
 <component typeCode="COMP">
     <ehrComposition classCode="COMPOSITION" moodCode="EVN">
         <id root="{{encounterStatementId}}" />
+        {{#altCode}}
+        {{{altCode}}}
+        {{/altCode}}
+        {{^altCode}}
         <code code="{{code}}" displayName="{{displayName}}" codeSystem="2.16.840.1.113883.2.1.3.2.4.15">
             {{#originalText}}<originalText>{{originalText}}</originalText>{{/originalText}}
         </code>
-        <statusCode code="{{status}}" />
+        {{/altCode}}
+        <statusCode code="{{{status}}}" />
         <effectiveTime>
             {{{effectiveTime}}}
         </effectiveTime>
         {{{availabilityTime}}}
         <author typeCode="AUT" contextControlCode="OP">
-            <time value="{{{authorTime}}}" />
+            <time value="{{authorTime}}" />
+            {{#agentRef}}
+            <agentRef classCode="AGNT">
+                <id root="{{agentRef}}" />
+            </agentRef>
+            {{/agentRef}}
+            {{^agentRef}}
```

Comment:
Why are these inverse selections needed? Are we not outputting an agentRef in both cases?